src/package/JUnitXml.TestLogger.nuspec

```diff
@@ -9,6 +9,7 @@
     <projectUrl>https://github.com/spekt/junit.testlogger</projectUrl>
     <description>Xml logger for JUnit v3 compliant xml report when test is running with "dotnet test" or "dotnet vstest".</description>
     <license type="expression">MIT</license>
+    <icon>packageIcon.png</icon>
   </metadata>
   <files>
     <file src="Microsoft.VisualStudio.TestPlatform.Extension.Junit.Xml.TestLogger.dll" target="build\_common\" />
```

Comment:
We need add a `<file src="packageIcon.png" target="" />` entry here to include the icon in the nupkg.

Can we check what is the best practice to place the icon? e.g. should the target be `content\images` or `images` instead of packge root?